README.md

```diff
@@ -207,6 +207,24 @@ Type reset to start again and test this by adding the phrase "to eat there...":
 
 # Troubleshooting
 
+* Deploy using Cloud Foundry 'cf push' gives:
+
+``FAILED
+Could not find service wcsi-conversation-service to bind to watson-conversation-slots-intro``
+
+You can still use `cf push`, but realize that it is trying to bind to the services listed in the manifest.yml
+
+So, there are 3 ways you can get this to work.
+
+    * remove the services listed in the manifest.
+    * change the names of your bluemix services to match the names in the manifest.
+    * change the names in the manifest to match the names of your bluemix services.
+
+The prefered  option is number 2. Changing the manifest means you have to change it again if you have to pull from master for any reason (bug fix, new feature, ...).
```

Comment:
The git pull stuff sounds ok, but I see that as more of a useful note than a reason to never edit your manifest. Git-savvy and manifest savvy devs would probably not be so opposed to tweaking the file locally and doing all the usual git stuff.